dm/dm/master/openapi.go

```diff
@@ -822,6 +823,95 @@ func (s *Server) DMAPIOperateTableStructure(c *gin.Context, taskName string, sou
 	}
 }
 
+// DMAPICreateTaskConfig create task_config_template url is: (POST /api/v1/task_configs/)
+func (s *Server) DMAPICreateTaskConfig(c *gin.Context) {
+	var req openapi.CreateOrUpdateTaskConfigRequest
+	if err := c.Bind(&req); err != nil {
+		_ = c.Error(err)
+		return
+	}
+	task := &req.Task
+	if err := task.Adjust(); err != nil {
+		_ = c.Error(err)
+		return
+	}
+	// prepare target db config
+	newCtx := c.Request.Context()
+	toDBCfg := config.GetTargetDBCfgFromOpenAPITask(task)
+	if adjustDBErr := adjustTargetDB(newCtx, toDBCfg); adjustDBErr != nil {
+		_ = c.Error(terror.WithClass(adjustDBErr, terror.ClassDMMaster))
+		return
+	}
+	if err := ha.PutOpenAPITaskConfig(s.etcdClient, *task, req.Overwrite); err != nil {
```

Comment:
So, does this mean we support change all the config fields without any restriction?🤔